src/DataCollection.WPF/DataCollection.WPF.csproj

```diff
@@ -9,13 +9,14 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Esri.ArcGISRuntime.ExampleApps.DataCollection.WPF</RootNamespace>
     <AssemblyName>DataCollection.WPF</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
Was this change intentional?

Reply:
Kinda, I was trying something out that I ended up not using. I'll revert.